src/mixins/createEvented.ts

```diff
@@ -119,8 +140,30 @@ const createEvented: EventedFactory = compose<EventedMixin, EventedOptions>({
 				method.call(this, event);
 			}
 		},
-		on(type: string, listener: EventedListener<Event>): Handle {
-			return on(listenersMap.get(this), type, resolveListener(listener));
+		on(...args: any[]): Handle {
+			const evented: Evented = this;
+			const listenerMap = listenersMap.get(evented);
+			if (args.length === 2) { /* overload: on(type, listener) */
+				let type: string;
+				let listeners: EventedListenerOrArray<TargettedEventObject>;
+				[ type, listeners ] = args;
+				if (Array.isArray(listeners)) {
+					const handles = listeners.map((listener) => on(listenerMap, type, resolveListener(listener)));
+					return handlesArraytoHandle(handles);
+				}
+				else {
+					return on(listenerMap, type, resolveListener(listeners));
+				}
+			}
+			else if (args.length === 1) { /* overload: on(listners) */
+				let listenerMapArg: EventedListenersMap;
+				[ listenerMapArg ] = args;
```

Comment:
I debated that, but it was inconsistent with above... I figured consistent style won out, but you are right, it is a bit silly.